app/controllers/api/v1/book_suggestions_controller.rb

```diff
@@ -1,17 +1,29 @@
 module Api
   module V1
     class BookSuggestionsController < ApiController
+      skip_before_action :authenticate_api_v1_user!, only: %i[new create_without_auth]
+
       def create
-        book_suggestion = BookSuggestion.new(book_suggestions_create_params)
-        book_suggestion.save!
-        render json: book_suggestion
+        create_new_book_suggestion
+      end
+
+      def create_without_auth
+        create_new_book_suggestion
       end
 
+      def new; end
```

Comment:
why do you need to define a new?

Reply:
Because it is an offense to rubocop to do it in more than one line.